elyra/pipeline/pipeline_definition.py

```diff
@@ -460,6 +468,27 @@ def get_node(self, node_id: str):
                     return node
         return None
 
+    def get_node_comments(self, node_id: str) -> Optional[str]:
+        """
+        Given a node id returns the assoicated comments in the pipeline
+        :param node_id: the node id
+        :return: the comments or None
+        """
+        comments = []
+
+        for pipeline in self.pipelines:
+            comment_list = pipeline.comments
+            for comment in comment_list:
+                assoicated_node_id_list = comment.get("associated_id_refs", [])
```

Comment:
Typo...
```suggestion
                associated_node_id_list = comment.get("associated_id_refs", [])
```
(and its reference below)